html/changelog.html

```diff
@@ -46,926 +46,6 @@
 		</td>
 	</tr>
 </table>
-
-			<h2 class="date">10 October 2022</h2>
```

Comment:
Did you mean to delete like 900 lines of changelog?

Reply:
lol, I did not, it's the bot